sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1532,35 +1532,10 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 					currentVersion = 201;
 					getDao(Contact.class).executeRaw("ALTER TABLE contacts ADD COLUMN additionalDetails varchar(512);");
 				case 202:
+					currentVersion = 202;
+					getDao(Sample.class).executeRaw("ALTER TABLE samples ADD COLUMN associatedcontact_id bigint REFERENCES contact (id);");
```

Comment:
Please use camel case in the mobile app (so => associatedContact_id) - SQLite requires this